task/07-yield-tasks.js

```diff
@@ -33,10 +33,24 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+    let line = true;
+    for(let i = 99; i>=0; --i ) {
+        if (line) { 
+            i<99 ? i+=1 : i;
```

Comment:
99% of the time you don't need to modify iterator variable outside of tihs line `for(let i = 99; i>=0; --i )`. This is one of the 99% cases